### PR TITLE
Council vote of account

### PR DESCRIPTION
**Pull request by ishrak-imam**
### Description

Fetch council votes of an account from the Litentry graph

### CODE REVIEW

**Comment by ishrak-imam** [comment]
Blocked by
https://github.com/litentry/litentry-graph/pull/161

**Review by Ekaanth — APPROVED** [review]
LGTM!!

File: src/ui/screens/Council/CandidateScreen.tsx [hunk 1]
```diff
@@ -153,8 +152,9 @@ export function CandidateScreen({route, navigation}: ScreenProps) {
 
 function Voter({account}: {account: string}) {
   const {data: accountInfo} = useAccount(account);
-  const {data: voterData} = useCouncilVotesOf(account);
-  const formatBalance = useFormatBalance();
+  const {data: councilVote} = useCouncilVotesOf(account);
+
+  console.log(councilVote);
```
Comment (silva-fj) [inline src/ui/screens/Council/CandidateScreen.tsx]: 👀

File: src/ui/screens/Council/CouncilScreen.tsx [hunk 2]
```diff
@@ -184,16 +185,17 @@ function CouncilVoteModal({visible, setVisible, candidates, module}: CouncilVote
   const [account, setAccount] = React.useState<string>();
   const [amount, setAmount] = React.useState<string>('');
   const [selectedCandidates, setSelectedCandidates] = React.useState<Array<string>>([]);
-  const {data: voterData} = useCouncilVotesOf(account);
+  const {data: councilVote} = useCouncilVotesOf(account);
+  const {theme} = useToggleTheme();
```
Comment (silva-fj) [inline src/ui/screens/Council/CouncilScreen.tsx]: You can also do: `const {dark} = useTheme();`

**Review by silva-fj — APPROVED** [review]

**Pull request merged**